instrumentation/runtime-telemetry/runtime-telemetry-java8/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java8/GarbageCollector.java

```diff
@@ -125,11 +127,23 @@ public void handleNotification(Notification notification, Object unused) {
 
       String gcName = notificationInfo.getGcName();
       String gcAction = notificationInfo.getGcAction();
+      String gcCause = notificationInfo.getGcCause();
       double duration = notificationInfo.getGcInfo().getDuration() / MILLIS_PER_S;
-
-      gcDuration.record(
-          duration,
-          Attributes.of(JvmAttributes.JVM_GC_NAME, gcName, JvmAttributes.JVM_GC_ACTION, gcAction));
+      boolean enableJvmGcCauseAttribute =
+          ConfigPropertiesUtil.getBoolean(
+              "otel.instrumentation.runtime-telemetry.jvm-gc-cause-attribute-enabled", false);
```

Comment:
Imo this isn't the correct way to read the configuration. Keep in mind that the possible configuration sources are different for agent, library and spring starter. You should handle this the same way as existing configuration options are handled.